Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -317,6 +365,10 @@ void AdvancedWidget::AddDescriptions()
   m_dump_xfb_target->SetDescription(tr(TR_DUMP_XFB_DESCRIPTION));
   m_disable_vram_copies->SetDescription(tr(TR_DISABLE_VRAM_COPIES_DESCRIPTION));
   m_use_fullres_framedumps->SetDescription(tr(TR_INTERNAL_RESOLUTION_FRAME_DUMPING_DESCRIPTION));
+  m_scaled_efb_exclude_slider->SetDescription(tr(TR_SCALED_EFB_EXCLUDE_DESCRIPTION));
+  m_scaled_efb_exclude_enable->SetDescription(tr(TR_SCALED_EFB_EXCLUDE_DESCRIPTION));
+  // TODO: Create GraphicsLabel or do something fitting current Dolphin standards
```

Comment:
No need for this TODO, we don't put tooltips on labels

Reply:
Removed